www/js/app.js

```diff
@@ -939,7 +939,9 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
         var listLength = dirEntryArray.length < MAX_SEARCH_RESULT_SIZE ? dirEntryArray.length : MAX_SEARCH_RESULT_SIZE;
         for (var i = 0; i < listLength; i++) {
             var dirEntry = dirEntryArray[i];
-            articleListDivHtml += '<a href="#" dirEntryId="' + dirEntry.toStringId().replace(/'/g, '&apos;') +
+            var dirEntryStringId = dirEntry.toStringId();
```

Comment:
My code proposal was intended to replace both lines. So this one should be removed now.

Reply:
Ya I have done that and am going to upload it....I just pressed the commit suggested changes and it did that.. sorry for that

Reply:
Please review now..I have done which you had suggested.